iptables/table.go

```diff
@@ -1014,14 +987,37 @@ func (t *Table) applyUpdates() error {
 		t.logCxt.WithField("iptablesInput", input).Debug("Writing to iptables")
 
 		var outputBuf, errBuf bytes.Buffer
-		cmd := t.newCmd(t.iptablesRestoreCmd, "--noflush", "--verbose")
+		args := []string{"--noflush", "--verbose"}
+		if features.RestoreSupportsLock {
+			// Versions of iptables-restore that support the xtables lock also make it mandatory.  Make sure
```

Comment:
By "mandatory", do you mean that iptables-restore will give up if the lock is not available immediately?  Because it obviously isn't mandatory in the sense that you still need args to tell it to retry...

Mostly just checking my understanding here; I can't think of an obvious wording improvement to the comment.

Reply:
I mean, it won't proceed to do any iptables programming without the lock.  Its default behaviour with no args is to quit immediately if it can't acquire the lock.

Will see if I can tighten up the wording.